package.json

```diff
@@ -17,6 +17,9 @@
     "pretty": "prettier --write ./",
     "update": "ncu -u",
     "update:url": "node _developer/updateDashboard.js",
+    "ssl:install": "mkcert -install",
+    "ssl:create": "node _developer/generateSSL.js",
+    "tunnel": "local-ssl-proxy --config proxy-config.json",
```

Comment:
Unnecessary complications. it's going to be really difficult to explain to people who don't understand what's happening under the hood and later for them to debug it.